sopel/plugin.py

```diff
@@ -615,25 +615,42 @@ def add_attribute(function):
     return add_attribute
 
 
-def intent(*intent_list):
-    """Decorate a callable to trigger on a message with any of the given intents.
+def ctcp(function=None, *command_list):
+    """Decorate a callable to trigger on CTCP commands (mostly, ``ACTION``).
 
-    :param str intent_list: one or more intent(s) on which to trigger (really,
-                            the only useful value is ``action``)
+    :param str ctcp_command: one or more CTCP command(s) on which to trigger
+                             (really, the only useful value is ``ACTION``)
```

Comment:
This is not true of CTCPs.

```suggestion
```

If you want to hint at what plugin devs might use the decorator for, I suggest:

```suggestion
                             (the most commonly useful value is ``ACTION``)
```